master.css

```diff
@@ -13,6 +13,18 @@ body {
 }
 table {
     margin-bottom: 2em; width: 100%;
+    border-spacing: 0;
+}
+thead {
+    position: -webkit-sticky;
+    position: -moz-sticky;
+    position: -ms-sticky;
+    position: -o-sticky;
```

Comment:
None of the three prefixes above exist. Only `sticky` (for Firefox) and `-webkit-sticky` (for Safari and iOS) are [actually implemented](http://caniuse.com/#feat=css-sticky).